charts/prometheus/templates/sts.yaml

```
@@ -96,7 +99,22 @@ spec:
           {{- if .Values.configmapReload.prometheus.containerPort }}
           ports:
             - containerPort: {{ .Values.configmapReload.prometheus.containerPort }}
+              {{- if .Values.configmapReload.prometheus.containerPortName }}
+              name: {{ .Values.configmapReload.prometheus.containerPortName }}
+              {{- end }}
+          {{- end }}
+          {{- with .Values.configmapReload.prometheus.livenessProbe }}
+          livenessProbe:
+            {{- toYaml . | nindent 12 }}
           {{- end }}
+          {{- with .Values.configmapReload.prometheus.readinessProbe }}
+          readinessProbe:
+            {{- toYaml . | nindent 12 }}
+          {{- end }}
+          {{- if .Values.configmapReload.prometheus.startupProbe }}
+          {{- $startupProbe := omit .Values.configmapReload.prometheus.startupProbe "enabled" }}
+          startupProbe:
+            {{- toYaml $startupProbe | nindent 12 }}
```

Closing the conditional has somehow slipped away.
```suggestion
            {{- toYaml $startupProbe | nindent 12 }}
          {{- end }}
```